README.md

```diff
@@ -1,5 +1,15 @@
 # LSIF development and testing utilities
 
+## Install
+
+Assumes a working Go installation:
+
+lsif-validate: `go get github.com/sourcegraph/lsif-test/cmd/lsif-validate`
+
+lsif-visualize: `go get github.com/sourcegraph/lsif-test/cmd/lsif-visualize`
```

Comment:
For copy-past friendliness :)
````suggestion
```
# lsif-validate
go get github.com/sourcegraph/lsif-test/cmd/lsif-validate

# lsif-visualize
go get github.com/sourcegraph/lsif-test/cmd/lsif-visualize
```
````

Reply:
Completely misread this and made a bad edit :roll_eyes: Gonna fix that, good idea :smile: